manifests/init.pp

```diff
@@ -21,7 +128,7 @@
   $config_dir_group    = 'root',
   $config_dir_mode     = '0755',
   $config_ensure       = 'file',
-  $config_path         = '/etc/eyaml/config.yaml',
```

Comment:
I made a change to make `$config_path = "${config_dir}/config.yaml"` I think it makes better sense now right?